tests/e2e/TimeoutSkipped/expected-output.txt

```diff
@@ -4,8 +4,8 @@ Killed by Test Framework: 2
 Killed by Static Analysis: 0
 Errored: 0
 Syntax Errors: 0
-Escaped: 2
-Timed Out: 1
```

Comment:
note for myself once this is ready for reaview (out from draft): this e2e test specifically was created to test `S`kipped mutants, so this change is incorrect IMO - we need to test `S`kipped timeout in this e2e test

Reply:
agree. actually we also discussed whether `sleep()` should be mutated at all which would bring another flavour to this test.